components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/LoginDialogFragment.kt

```diff
@@ -128,7 +136,14 @@ internal class LoginDialogFragment : PromptDialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         view.findViewById<AppCompatTextView>(R.id.host_name).text = origin
-
+        val iconView = view.findViewById<ImageView>(R.id.host_icon)
+        if (icon != null) {
+            iconView.setImageBitmap(icon)
```

Comment:
nit: We could do `iconView?.setImageBitmap(icon)`

Reply:
@Amejia481 well we could, but we still need an else statement, since the tinting should happen only for the default icon

Reply:
👍